src/utils/checkOrResizeImage.spec.js

```diff
@@ -0,0 +1,28 @@
+
+describe('checkOrResizeImage test', () => {
+  const canvas = document.createElement('canvas');
+
```

Comment:
this function  **checkOrResizeImage**  draws canvas
as to google search - the tests for this function requires  **jest-canvas-mock** dev dependancy
can i use it or how should i test instead ?